packages/amplication-notifications-service/src/providers/secrets/base/secretsManager.service.base.ts

```diff
@@ -0,0 +1,19 @@
+import { ConfigService } from '@nestjs/config';
+
+export interface ISecretsManager {
+  getSecret: (key: string) => Promise<any | null>;
+}
+
+export class SecretsManagerServiceBase implements ISecretsManager {
+  constructor(protected readonly configService: ConfigService) {}
+  async getSecret<T>(key: string): Promise<T | null> {
+    if (!key) {
+      throw new Error("Didn't got the key");
```

Comment:
This also is incorrect grammatically. On top of that, there is not enough actionable information in this error message for us to even understand what went wrong. May I suggest:

```
throw new Error(`Key "${key}" could not be found`);
```

Reply:
this is an auto-generated file by Amplicaton, but I will fix it (we should fix it also in the generated app)
There is also a grammar issue with "didn't got"😣